--- a/com/pixelmonmod/pixelmon/client/gui/widgets/PokeCheckerButton.java
+++ b/com/pixelmonmod/pixelmon/client/gui/widgets/PokeCheckerButton.java
@@ -83,9 +83,7 @@
             this.summary.render(matrix, mouseX, mouseY, partialTicks);
             this.moves.render(matrix, mouseX, mouseY, partialTicks);
             this.stats.render(matrix, mouseX, mouseY, partialTicks);
-            ScreenHelper.drawCenteredString(
-                matrix, this.pokemon.getDisplayName(), (float)this.x + (float)this.width / 2.0F, (float)(this.y + 2), 16777215, true
-            );
+         ScreenHelper.drawCenteredString(matrix, this.pokemon.getDisplayName(), (float)this.x + (float)this.width / 2.0F, (float)(this.y + 2), 16777215, true);
             this.summary.drawText(matrix);
             this.moves.drawText(matrix);
             this.stats.drawText(matrix);
