--- a/com/pixelmonmod/tcg/duel/log/parameters/DuelLogAttackParameters.java
+++ b/com/pixelmonmod/tcg/duel/log/parameters/DuelLogAttackParameters.java
@@ -36,7 +36,7 @@
         this.damage = buf.readInt();
         if (buf.readBoolean()) {
             int count = buf.readInt();
-            this.conditions = new ArrayList();
+         this.conditions = new ArrayList<>();
 
             for(int i = 0; i < count; ++i) {
                 CardCondition cardCondition = CardCondition.values()[buf.readInt()];
@@ -45,7 +45,7 @@
                     modifier = buf.readInt();
                 }
 
-                this.conditions.add(new ImmutablePair(cardCondition, modifier));
+            this.conditions.add(new ImmutablePair<>(cardCondition, modifier));
             }
         } else {
             this.conditions = null;
