--- a/com/pixelmonmod/pixelmon/client/render/player/RenderEvents.java
+++ b/com/pixelmonmod/pixelmon/client/render/player/RenderEvents.java
@@ -97,9 +97,7 @@
                                 );
                             }
                         );
-                    UIElementItem.getText(stack, hover)
-                        .ifPresent(
-                            text -> {
+               UIElementItem.getText(stack, hover).ifPresent(text -> {
                                 TextJustification j = UIElementItem.getTextJustification(stack);
                                 float s = (float)UIElementItem.getTextScale(stack);
                                 switch(j) {
@@ -107,17 +105,12 @@
                                         ScreenHelper.drawScaledString(matrix, text, 0.0F, 0.0F, new Color(rgba[0], rgba[1], rgba[2], rgba[3]).getRGB(), s);
                                         break;
                                     case CENTER:
-                                        ScreenHelper.drawScaledCenteredString(
-                                            matrix, text, 0.0F, 0.0F, new Color(rgba[0], rgba[1], rgba[2], rgba[3]).getRGB(), s
-                                        );
+                        ScreenHelper.drawScaledCenteredString(matrix, text, 0.0F, 0.0F, new Color(rgba[0], rgba[1], rgba[2], rgba[3]).getRGB(), s);
                                         break;
                                     case LEFT:
-                                        ScreenHelper.drawScaledStringRightAligned(
-                                            matrix, text, 0.0F, 0.0F, new Color(rgba[0], rgba[1], rgba[2], rgba[3]).getRGB(), false, s
-                                        );
+                        ScreenHelper.drawScaledStringRightAligned(matrix, text, 0.0F, 0.0F, new Color(rgba[0], rgba[1], rgba[2], rgba[3]).getRGB(), false, s);
                                 }
-                            }
-                        );
+               });
                     matrix.popPose();
                 }
             }
