--- a/com/pixelmonmod/pixelmon/command/impl/NpcKillCommand.java
+++ b/com/pixelmonmod/pixelmon/command/impl/NpcKillCommand.java
@@ -33,7 +33,7 @@
         int count = 0;
         int trainersInBattle = 0;
 
-        for(Entity entity : (List)world.getEntities().collect(Collectors.toList())) {
+      for(Entity entity : world.getEntities().collect(Collectors.toList())) {
             if (entity instanceof NPCEntity) {
                 NPCEntity npc = (NPCEntity)entity;
                 if (npc instanceof NPCTrainer) {
