--- a/com/pixelmonmod/pixelmon/quests/client/ui/GuiListQuest.java
+++ b/com/pixelmonmod/pixelmon/quests/client/ui/GuiListQuest.java
@@ -130,10 +130,7 @@
     }
 
     public QuestProgressClient mouseClicked(double mouseX, double mouseY, int mouseButton, int offsetY) {
-        if (mouseX > (double)this.xPos
-            && mouseX < (double)(this.xPos + this.width)
-            && mouseY > (double)this.yPos
-            && mouseY < (double)(this.yPos + this.height)) {
+      if (mouseX > (double)this.xPos && mouseX < (double)(this.xPos + this.width) && mouseY > (double)this.yPos && mouseY < (double)(this.yPos + this.height)) {
             mouseY += 8.0;
             int theLine = (int)((mouseY - (double)offsetY) / (double)this.lineHeight);
             int index = theLine + this.startLine;
