--- a/com/pixelmonmod/tcg/duel/attack/effects/FlipsMultiplyEffect.java
+++ b/com/pixelmonmod/tcg/duel/attack/effects/FlipsMultiplyEffect.java
@@ -20,7 +20,7 @@
 
     @Override
     public List<CoinSide> flipCoin(List<CoinSide> currentResult, PokemonCardState pokemon, GameServerState server) {
-        List<CoinSide> var6 = new ArrayList();
+      List<CoinSide> var6 = new ArrayList<>();
         int count = this.getCount(pokemon, server);
         if (count == -1) {
             while(true) {
