--- a/com/pixelmonmod/pixelmon/api/config/api/data/Serializers.java
+++ b/com/pixelmonmod/pixelmon/api/config/api/data/Serializers.java
@@ -1,6 +1,6 @@
 package com.pixelmonmod.pixelmon.api.config.api.data;
 
-import info.pixelmon.repack.org.spongepowered.serialize.ScalarSerializer;
+import org.spongepowered.configurate.serialize.ScalarSerializer;
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
