--- a/com/pixelmonmod/pixelmon/battles/attacks/specialAttacks/basic/Thief.java
+++ b/com/pixelmonmod/pixelmon/battles/attacks/specialAttacks/basic/Thief.java
@@ -38,10 +38,7 @@
     ) {
         if (!userChoice.hitsAlly()) {
             for(PixelmonWrapper target : userChoice.targets) {
-                if (!pw.hasHeldItem()
-                    && target.hasHeldItem()
-                    && !target.getHeldItem().hasNegativeEffect()
-                    && !(target.getBattleAbility() instanceof StickyHold)) {
+            if (!pw.hasHeldItem() && target.hasHeldItem() && !target.getHeldItem().hasNegativeEffect() && !(target.getBattleAbility() instanceof StickyHold)) {
                     userChoice.raiseWeight(25.0F);
                 }
             }
