--- a/com/pixelmonmod/pixelmon/api/replacement/ReplacementLogicRegistry.java
+++ b/com/pixelmonmod/pixelmon/api/replacement/ReplacementLogicRegistry.java
@@ -22,7 +22,7 @@
     }
 
     public static void register(Supplier<? extends SpawnReplacementLogic> replacementLogic) {
-        SPAWN_REPLACEMENT_LOGIC.put(replacementLogic.get().getId().toLowerCase(), replacementLogic);
+      SPAWN_REPLACEMENT_LOGIC.put(replacementLogic.get().getId().toLowerCase(), (Supplier<SpawnReplacementLogic>) replacementLogic);
     }
 
     static {
