--- a/com/pixelmonmod/pixelmon/blocks/BauxiteOreBlock.java
+++ b/com/pixelmonmod/pixelmon/blocks/BauxiteOreBlock.java
@@ -9,12 +9,7 @@
 public class BauxiteOreBlock extends Block {
     public BauxiteOreBlock() {
         super(
-            Properties.of(Material.STONE)
-                .strength(5.0F, 0.0F)
-                .harvestLevel(2)
-                .harvestTool(ToolType.PICKAXE)
-                .requiresCorrectToolForDrops()
-                .sound(SoundType.STONE)
+         Properties.of(Material.STONE).strength(5.0F, 0.0F).harvestLevel(2).harvestTool(ToolType.PICKAXE).requiresCorrectToolForDrops().sound(SoundType.STONE)
         );
     }
 }
