--- a/com/pixelmonmod/pixelmon/blocks/furniture/BenchBlock.java
+++ b/com/pixelmonmod/pixelmon/blocks/furniture/BenchBlock.java
@@ -64,12 +64,7 @@
             for(Entity entity : world.getEntitiesOfClass(
                 Entity.class,
                 new AABB(
-                    (double)pos.getX(),
-                    (double)(pos.getY() - 1),
-                    (double)pos.getZ(),
-                    (double)(pos.getX() + 1),
-                    (double)(pos.getY() + 2),
-                    (double)(pos.getZ() + 1)
+               (double)pos.getX(), (double)(pos.getY() - 1), (double)pos.getZ(), (double)(pos.getX() + 1), (double)(pos.getY() + 2), (double)(pos.getZ() + 1)
                 )
             )) {
                 if (entity instanceof ChairMountEntity) {
