--- a/com/pixelmonmod/pixelmon/client/particle/ParticleArcanery.java
+++ b/com/pixelmonmod/pixelmon/client/particle/ParticleArcanery.java
@@ -42,7 +42,7 @@
             return 0.0;
         }
     });
-    private static final List<ParticleArcanery> PARTICLE_QUEUE = new ArrayList();
+   private static final List<ParticleArcanery> PARTICLE_QUEUE = new ArrayList<>();
     public ParticleEffect effect;
     private float partialTicks;
     public float rotationX;
