--- a/com/pixelmonmod/pixelmon/blocks/tileentity/GracideaTileEntity.java
+++ b/com/pixelmonmod/pixelmon/blocks/tileentity/GracideaTileEntity.java
@@ -58,7 +58,7 @@
                 return (float)this.level.getBrightness(LightLayer.SKY, posx) < 14.0F;
             }
         });
-        ArrayList<BlockPos> chosen = new ArrayList();
+      ArrayList<BlockPos> chosen = new ArrayList<>();
 
         for(int i = 0; i < numToGrow; ++i) {
             if (dirs.size() > 1) {
