--- a/com/pixelmonmod/pixelmon/client/gui/pokechecker/PokeCheckerWarningDeleteMoveScreen.java
+++ b/com/pixelmonmod/pixelmon/client/gui/pokechecker/PokeCheckerWarningDeleteMoveScreen.java
@@ -26,10 +26,8 @@
     @Override
     protected void drawWarningText(PoseStack matrix) {
         drawCenteredString(matrix, this.minecraft.font, I18n.get("gui.screenpokechecker.forget1", new Object[0]), 60, 68, 16777215);
+      drawCenteredString(matrix, this.minecraft.font, I18n.get("gui.screenpokechecker.forget2", new Object[]{this.pokemon.getDisplayName()}), 60, 78, 16777215);
         drawCenteredString(
-            matrix, this.minecraft.font, I18n.get("gui.screenpokechecker.forget2", new Object[]{this.pokemon.getDisplayName()}), 60, 78, 16777215
-        );
-        drawCenteredString(
             matrix, this.minecraft.font, this.pokemon.getMoveset().get(this.moveToDelete).getActualMove().getLocalizedName() + "?", 60, 88, 16777215
         );
     }
