--- a/com/pixelmonmod/pixelmon/battles/raids/RaidSettings.java
+++ b/com/pixelmonmod/pixelmon/battles/raids/RaidSettings.java
@@ -167,7 +167,7 @@
     }
 
     public ArrayList<ItemStack> getDrops(List<Element> types) {
-        ArrayList<ItemStack> stacks = new ArrayList();
+      ArrayList<ItemStack> stacks = new ArrayList<>();
 
         for(int i = 0; i < this.rewardCount; ++i) {
             ItemStack stack = this.potentialRewards.get();
