--- a/com/pixelmonmod/pixelmon/quests/client/editor/ui/ListText.java
+++ b/com/pixelmonmod/pixelmon/quests/client/editor/ui/ListText.java
@@ -50,14 +50,7 @@
             float barMult = 9.0F / ((float)this.lines.size() - 8.0F);
             float barHeight = (float)this.lineHeight * barMult;
             float barPos = barHeight * (float)this.startLine;
-            fill(
-                matrix,
-                this.xPos + this.width,
-                this.yPos + 3 + (int)barPos,
-                this.xPos + this.width + 6,
-                this.yPos + 3 + (int)barPos + (int)barHeight,
-                -6710887
-            );
+         fill(matrix, this.xPos + this.width, this.yPos + 3 + (int)barPos, this.xPos + this.width + 6, this.yPos + 3 + (int)barPos + (int)barHeight, -6710887);
             fill(matrix, this.xPos + this.width, this.yPos + 1, this.xPos + this.width + 6, this.yPos + 3, -7829368);
             fill(matrix, this.xPos + this.width, this.yPos + 1 + bottomMarker, this.xPos + this.width + 6, this.yPos + 3 + bottomMarker, -7829368);
         }
