--- a/com/pixelmonmod/pixelmon/battles/animations/particles/ParticleSmoke.java
+++ b/com/pixelmonmod/pixelmon/battles/animations/particles/ParticleSmoke.java
@@ -35,9 +35,7 @@
             this.spriteSet = spriteSet;
         }
 
-        public Particle createParticle(
-            SimpleParticleType typeIn, ClientLevel worldIn, double x, double y, double z, double xSpeed, double ySpeed, double zSpeed
-        ) {
+      public Particle createParticle(SimpleParticleType typeIn, ClientLevel worldIn, double x, double y, double z, double xSpeed, double ySpeed, double zSpeed) {
             return new ParticleSmoke(worldIn, x, y, z, xSpeed, ySpeed, zSpeed, this.spriteSet);
         }
     }
