--- a/com/pixelmonmod/pixelmon/client/gui/npcEditor/ChattingNPCEditorScreen.java
+++ b/com/pixelmonmod/pixelmon/client/gui/npcEditor/ChattingNPCEditorScreen.java
@@ -61,9 +61,7 @@
             new IndexedButton(1, this.width / 2 + 155, this.height / 2 + 90, 30, 20, new TranslatableComponent("gui.guiItemDrops.ok"), this::actionPerformed)
         );
         this.addButton(
-            new IndexedButton(
-                6, this.width / 2 + 100, this.height / 2 - 120, 80, 20, new TranslatableComponent("gui.npceditor.despawn"), this::actionPerformed
-            )
+         new IndexedButton(6, this.width / 2 + 100, this.height / 2 - 120, 80, 20, new TranslatableComponent("gui.npceditor.despawn"), this::actionPerformed)
         );
         this.nameBox = new EditBox(this.minecraft.font, this.width / 2 - 20, this.height / 2 - 96, 180, 20, TextComponent.EMPTY);
         this.nameBox.setValue(name);
@@ -92,9 +90,7 @@
                     npc -> {
                         if (npc.getID().startsWith("Custom")) {
                             NetworkHelper.sendToServer(
-                                new NPCServerPacket(
-                                    currentNPCID, EnumNPCServerPacketType.TextureIndex, this.npc.getBaseTrainer().textures.indexOf(npc.getID())
-                                )
+                        new NPCServerPacket(currentNPCID, EnumNPCServerPacketType.TextureIndex, this.npc.getBaseTrainer().textures.indexOf(npc.getID()))
                             );
                             this.textureBox.setValue("");
                         } else {
@@ -145,20 +141,12 @@
         );
         ScreenHelper.drawEntity(matrix, this.npc, this.width / 2 - 140, this.height / 2 + 50, 60.0F, 0.0F, 0.0F);
         this.minecraft.font.draw(matrix, I18n.get("gui.trainereditor", new Object[0]), (float)this.width / 2.0F - 60.0F, (float)this.height / 2.0F - 90.0F, 0);
+      this.minecraft.font.draw(matrix, I18n.get("gui.npceditor.page1", new Object[0]), (float)this.width / 2.0F - 80.0F, (float)this.height / 2.0F - 61.0F, 0);
+      this.minecraft.font.draw(matrix, I18n.get("gui.npceditor.page2", new Object[0]), (float)this.width / 2.0F - 80.0F, (float)this.height / 2.0F - 26.0F, 0);
+      this.minecraft.font.draw(matrix, I18n.get("gui.npceditor.page3", new Object[0]), (float)this.width / 2.0F - 80.0F, (float)this.height / 2.0F + 9.0F, 0);
+      this.minecraft.font.draw(matrix, I18n.get("gui.npceditor.page4", new Object[0]), (float)this.width / 2.0F - 80.0F, (float)this.height / 2.0F + 44.0F, 0);
         this.minecraft
             .font
-            .draw(matrix, I18n.get("gui.npceditor.page1", new Object[0]), (float)this.width / 2.0F - 80.0F, (float)this.height / 2.0F - 61.0F, 0);
-        this.minecraft
-            .font
-            .draw(matrix, I18n.get("gui.npceditor.page2", new Object[0]), (float)this.width / 2.0F - 80.0F, (float)this.height / 2.0F - 26.0F, 0);
-        this.minecraft
-            .font
-            .draw(matrix, I18n.get("gui.npceditor.page3", new Object[0]), (float)this.width / 2.0F - 80.0F, (float)this.height / 2.0F + 9.0F, 0);
-        this.minecraft
-            .font
-            .draw(matrix, I18n.get("gui.npceditor.page4", new Object[0]), (float)this.width / 2.0F - 80.0F, (float)this.height / 2.0F + 44.0F, 0);
-        this.minecraft
-            .font
             .draw(matrix, I18n.get("gui.trainereditor.model", new Object[0]), (float)this.width / 2.0F - 180.0F, (float)this.height / 2.0F + 72.0F, 0);
     }
 
