--- a/com/pixelmonmod/pixelmon/client/gui/npcEditor/TrainerEditorMoreScreen.java
+++ b/com/pixelmonmod/pixelmon/client/gui/npcEditor/TrainerEditorMoreScreen.java
@@ -42,7 +42,7 @@
     protected int listLeft;
     protected int listHeight;
     protected int listWidth;
-    ArrayList<ItemStack> dropList = new ArrayList();
+   ArrayList<ItemStack> dropList = new ArrayList<>();
     TrainerDropListSlotScreen list;
     int lastWidth;
     int lastHeight;
@@ -82,9 +82,7 @@
         this.initList();
         this.children
             .add(
-                new IndexedButton(
-                    1, this.width / 2 + 155, this.height / 2 + 90, 30, 20, new TranslatableComponent("gui.guiItemDrops.ok"), this::actionPerformed
-                )
+            new IndexedButton(1, this.width / 2 + 155, this.height / 2 + 90, 30, 20, new TranslatableComponent("gui.guiItemDrops.ok"), this::actionPerformed)
             );
         this.children
             .add(
