--- a/com/pixelmonmod/pixelmon/api/pokemon/lure/SteadyLureExpiring.java
+++ b/com/pixelmonmod/pixelmon/api/pokemon/lure/SteadyLureExpiring.java
@@ -28,8 +28,7 @@
                         if (lure.getDamageValue() >= 128) {
                             LureExpiredEvent expiredEvent = new LureExpiredEvent(player);
                             if (!Pixelmon.EVENT_BUS.post(expiredEvent)) {
-                                player.level
-                                    .playSound(null, player.getX(), player.getY(), player.getZ(), SoundEvents.LAVA_POP, SoundSource.AMBIENT, 1.0F, 1.5F);
+                        player.level.playSound(null, player.getX(), player.getY(), player.getZ(), SoundEvents.LAVA_POP, SoundSource.AMBIENT, 1.0F, 1.5F);
                                 party.setLure(null);
                                 player.sendMessage(new TranslatableComponent("pixelmon.lure.expire"), Util.NIL_UUID);
                                 if (expiredEvent.casingStack != null && !expiredEvent.casingStack.isEmpty()) {
