--- a/com/pixelmonmod/pixelmon/client/render/layers/LayerSash.java
+++ b/com/pixelmonmod/pixelmon/client/render/layers/LayerSash.java
@@ -122,9 +122,7 @@
                 this.sashBG
                     .getPixels()
                     .setPixelRGBA(
-                        0,
-                        0,
-                        0xFF000000 | (int)((double)(r * 255.0F) + 0.5) << 16 | (int)((double)(g * 255.0F) + 0.5) << 8 | (int)((double)(b * 255.0F) + 0.5)
+                  0, 0, 0xFF000000 | (int)((double)(r * 255.0F) + 0.5) << 16 | (int)((double)(g * 255.0F) + 0.5) << 8 | (int)((double)(b * 255.0F) + 0.5)
                     );
                 this.sashBG.upload();
                 this.SASH_MODEL.getModel().getModel().animate();
