--- a/com/pixelmonmod/pixelmon/client/gui/widgets/text/EnhancedTextFieldWidget.java
+++ b/com/pixelmonmod/pixelmon/client/gui/widgets/text/EnhancedTextFieldWidget.java
@@ -10,7 +10,7 @@
 
 @OnlyIn(Dist.CLIENT)
 public class EnhancedTextFieldWidget extends EditBox {
-    protected static final Pattern COLOR_CODE_PATTERN = Pattern.compile("[&\u00a7][0-9a-fk-orA-FK-OR]");
+   protected static final Pattern COLOR_CODE_PATTERN = Pattern.compile("[&§][0-9a-fk-orA-FK-OR]");
     protected final Font fontRenderer;
     protected final int field_230690_l_;
     protected final int field_230691_m_;
