--- a/com/pixelmonmod/pixelmon/client/gui/pokemoneditor/IndividualEditorBaseScreen.java
+++ b/com/pixelmonmod/pixelmon/client/gui/pokemoneditor/IndividualEditorBaseScreen.java
@@ -67,9 +67,7 @@
         this.buttons.clear();
         this.buttons
             .add(
-                new IndexedButton(
-                    1, this.width / 2 + 155, this.height / 2 + 90, 30, 20, new TranslatableComponent("gui.guiItemDrops.ok"), this::actionPerformed
-                )
+            new IndexedButton(1, this.width / 2 + 155, this.height / 2 + 90, 30, 20, new TranslatableComponent("gui.guiItemDrops.ok"), this::actionPerformed)
             );
         List<PaletteProperties> shinyOptions = Lists.newArrayList(this.p.getForm().getGenderProperties(this.p.getGender()).getPalettes());
         Species species = this.p.getSpecies();
@@ -85,13 +83,7 @@
         this.buttons
             .add(
                 this.genderButton = new IndexedButton(
-                    3,
-                    this.width / 2 - 120,
-                    this.height / 2 + 30,
-                    80,
-                    20,
-                    I18n.get(this.p.getGender().getTranslationKey(), new Object[0]),
-                    this::actionPerformed
+               3, this.width / 2 - 120, this.height / 2 + 30, 80, 20, I18n.get(this.p.getGender().getTranslationKey(), new Object[0]), this::actionPerformed
                 )
             );
         this.addDropDown(
@@ -124,8 +116,8 @@
             .setCompletions(PixelmonSpecies.getAll());
         this.tbName.setValue(this.p.getSpecies());
         this.tbName.setResponder(s -> {
-            if (PixelmonSpecies.get(s.replace("\u00e9", "e")).isPresent()) {
-                Species newSpecies = PixelmonSpecies.get(s.replace("\u00e9", "e")).get().getValueUnsafe();
+         if (PixelmonSpecies.get(s.replace("é", "e")).isPresent()) {
+            Species newSpecies = PixelmonSpecies.get(s.replace("é", "e")).get().getValueUnsafe();
                 if (newSpecies != null && !this.tbName.getTranslatableValue().is(newSpecies)) {
                     this.tbName.setValue(newSpecies);
                     this.changePokemon(this.tbName.getTranslatableValue());
@@ -179,13 +171,7 @@
             this.buttons
                 .add(
                     new IndexedButton(
-                        14,
-                        this.width / 2 - 185,
-                        this.height / 2 - 110,
-                        90,
-                        20,
-                        new TranslatableComponent("gui.trainereditor.deletepoke"),
-                        this::actionPerformed
+                  14, this.width / 2 - 185, this.height / 2 - 110, 90, 20, new TranslatableComponent("gui.trainereditor.deletepoke"), this::actionPerformed
                     )
                 );
         }
@@ -230,8 +216,7 @@
     }
 
     private int getLongestString(List<? extends ITranslatable> items) {
-        return ScreenHelper.getLongestStringWidth(items.stream().filter(Objects::nonNull).map(ITranslatable::getLocalizedName).collect(Collectors.toList()))
-            + 2;
+      return ScreenHelper.getLongestStringWidth(items.stream().filter(Objects::nonNull).map(ITranslatable::getLocalizedName).collect(Collectors.toList())) + 2;
     }
 
     protected EditBox createExtraTextField() {
@@ -272,17 +257,11 @@
         this.minecraft
             .font
             .draw(
-                matrix,
-                this.titleText,
-                (float)this.width / 2.0F - (float)this.minecraft.font.width(this.titleText) / 2.0F,
-                (float)this.height / 2.0F - 90.0F,
-                0
+            matrix, this.titleText, (float)this.width / 2.0F - (float)this.minecraft.font.width(this.titleText) / 2.0F, (float)this.height / 2.0F - 90.0F, 0
             );
+      this.minecraft.font.draw(matrix, I18n.get("gui.trainereditor", new Object[0]), (float)this.width / 2.0F - 180.0F, (float)this.height / 2.0F - 45.0F, 0);
         this.minecraft
             .font
-            .draw(matrix, I18n.get("gui.trainereditor", new Object[0]), (float)this.width / 2.0F - 180.0F, (float)this.height / 2.0F - 45.0F, 0);
-        this.minecraft
-            .font
             .draw(matrix, I18n.get("gui.trainereditor.lvl", new Object[0]), (float)this.width / 2.0F - 180.0F, (float)this.height / 2.0F - 25.0F, 0);
         this.minecraft
             .font
@@ -415,9 +394,7 @@
                 }
 
                 button.setMessage(
-                    this.p.getGender() == Gender.MALE
-                        ? new TranslatableComponent("gui.trainereditor.male")
-                        : new TranslatableComponent("gui.trainereditor.female")
+               this.p.getGender() == Gender.MALE ? new TranslatableComponent("gui.trainereditor.male") : new TranslatableComponent("gui.trainereditor.female")
                 );
             } else if (id == 6) {
                 if (this.checkFields()) {
