--- a/com/pixelmonmod/pixelmon/battles/status/AuroraVeil.java
+++ b/com/pixelmonmod/pixelmon/battles/status/AuroraVeil.java
@@ -40,12 +40,7 @@
 
     public AuroraVeil(PixelmonWrapper user, int turns) {
         super(
-            StatusType.AuroraVeil,
-            null,
-            turns,
-            "pixelmon.effect.auroraveil.raised",
-            "pixelmon.effect.auroraveil.already",
-            "pixelmon.status.auroraveil.woreoff"
+         StatusType.AuroraVeil, null, turns, "pixelmon.effect.auroraveil.raised", "pixelmon.effect.auroraveil.already", "pixelmon.status.auroraveil.woreoff"
         );
         this.user = user;
     }
