--- a/com/pixelmonmod/pixelmon/api/config/adapter/PokeLootRateAdapter.java
+++ b/com/pixelmonmod/pixelmon/api/config/adapter/PokeLootRateAdapter.java
@@ -1,8 +1,8 @@
 package com.pixelmonmod.pixelmon.api.config.adapter;
 
 import com.pixelmonmod.pixelmon.api.config.PokeLootRate;
-import info.pixelmon.repack.org.spongepowered.serialize.ScalarSerializer;
-import info.pixelmon.repack.org.spongepowered.serialize.SerializationException;
+import org.spongepowered.configurate.serialize.ScalarSerializer;
+import org.spongepowered.configurate.serialize.SerializationException;
 import java.lang.reflect.Type;
 import java.util.function.Predicate;
 
