--- a/com/pixelmonmod/tcg/TCG.java
+++ b/com/pixelmonmod/tcg/TCG.java
@@ -67,7 +67,7 @@
                                 && !serverVersion.equalsIgnoreCase(NetworkRegistry.ACCEPTVANILLA)
                             ? "9.1.2".equals(serverVersion)
                             : true,
-                    "9.1.2"::equals
+                        (Predicate<String>) "9.1.2"::equals
                 )
             );
         } catch (InvocationTargetException | IllegalAccessException | NoSuchMethodException var2) {
