--- a/com/pixelmonmod/pixelmon/world/dimension/client/PixelmonDimensionRenderInfo.java
+++ b/com/pixelmonmod/pixelmon/world/dimension/client/PixelmonDimensionRenderInfo.java
@@ -466,16 +466,13 @@
                             && !CampfireBlock.isLitCampfire(blockstate)
                         ? ParticleTypes.RAIN
                         : ParticleTypes.SMOKE;
-                    world.addParticle(
-                        iparticledata, (double)blockpos2.getX() + d0, (double)blockpos2.getY() + d4, (double)blockpos2.getZ() + d1, 0.0, 0.0, 0.0
-                    );
+               world.addParticle(iparticledata, (double)blockpos2.getX() + d0, (double)blockpos2.getY() + d4, (double)blockpos2.getZ() + d1, 0.0, 0.0, 0.0);
                 }
             }
 
             if (blockpos1 != null && random.nextInt(3) < renderer.rainSoundTime++) {
                 renderer.rainSoundTime = 0;
-                if (blockpos1.getY() > blockpos.getY() + 1
-                    && world.getHeightmapPos(Types.MOTION_BLOCKING, blockpos).getY() > Mth.floor((float)blockpos.getY())) {
+            if (blockpos1.getY() > blockpos.getY() + 1 && world.getHeightmapPos(Types.MOTION_BLOCKING, blockpos).getY() > Mth.floor((float)blockpos.getY())) {
                     world.playLocalSound(blockpos1, SoundEvents.WEATHER_RAIN_ABOVE, SoundSource.WEATHER, 0.1F, 0.5F, false);
                 } else {
                     world.playLocalSound(blockpos1, SoundEvents.WEATHER_RAIN, SoundSource.WEATHER, 0.2F, 1.0F, false);
