--- a/com/pixelmonmod/pixelmon/quests/objectives/objectives/inserters/InsertConcatenationObjective.java
+++ b/com/pixelmonmod/pixelmon/quests/objectives/objectives/inserters/InsertConcatenationObjective.java
@@ -24,10 +24,7 @@
     @Override
     public Arguments parse(Quest quest, Stage stage, ArgsIn args) {
         return Arguments.create(
-            Argument.from(args.get(0), s -> s),
-            Argument.from(args.get(1), s -> s),
-            Argument.from(args.get(2), s -> s),
-            Argument.from(args.get(3), s -> s, " ")
+         Argument.from(args.get(0), s -> s), Argument.from(args.get(1), s -> s), Argument.from(args.get(2), s -> s), Argument.from(args.get(3), s -> s, " ")
         );
     }
 
