--- a/com/pixelmonmod/pixelmon/api/spawning/archetypes/entities/wormholes/SpawnActionWormhole.java
+++ b/com/pixelmonmod/pixelmon/api/spawning/archetypes/entities/wormholes/SpawnActionWormhole.java
@@ -16,11 +16,7 @@
             (double)this.spawnLocation.location.pos.getY(),
             (double)this.spawnLocation.location.pos.getZ(),
             ((SpawnInfoWormhole)this.spawnInfo).minMaxAge
-                + this.spawnLocation
-                    .location
-                    .world
-                    .random
-                    .nextInt(((SpawnInfoWormhole)this.spawnInfo).maxMaxAge - ((SpawnInfoWormhole)this.spawnInfo).minMaxAge)
+            + this.spawnLocation.location.world.random.nextInt(((SpawnInfoWormhole)this.spawnInfo).maxMaxAge - ((SpawnInfoWormhole)this.spawnInfo).minMaxAge)
         );
     }
 }
